tests/baselines/reference/classMemberInitializerScoping2(target=es2017,usedefineforclassfields=true).errors.txt

```diff
@@ -1,12 +0,0 @@
-classMemberInitializerScoping2.ts(3,9): error TS2301: Initializer of instance member variable 'p' cannot reference identifier 'x' declared in the constructor.
-
-
-==== classMemberInitializerScoping2.ts (1 errors) ====
-    const x = 1
-    class C {
-        p = x
-            ~
-!!! error TS2301: Initializer of instance member variable 'p' cannot reference identifier 'x' declared in the constructor.

```

Comment:
This test has the combo of es2017/esnext and useDefineForClassFields=true/false. Before, es2017 always errored, and esnext only errored when useDefineForClassFields=false.

Now, es2017 with useDefineForClassFields=false also no longer errors.

Is that correct? I'm not totally sure. @sandersn